src/block-data-providers/esplora.provider.ts

```diff
@@ -258,4 +224,8 @@ export class EsploraProvider
             url: `${this.baseUrl}/tx/${txid}`,
         });
     }
+
+    private async request(request_config: AxiosRequestConfig): Promise<any> {
+        return makeRequest(request_config, this.retryConfig, this.logger);
+    }
```

Comment:
Instead of this, we should directly call `makeRequest` wherever `request` is called